plugins/EcampCourseAim/src/EcampCourseAim/Entity/Item.php

```diff
@@ -43,7 +42,7 @@ class Item extends BaseEntity
      *      inverseJoinColumns={@ORM\JoinColumn(name="EventPlugin_id", referencedColumnName="id", onDelete="cascade")}
      *      )
      */
-    private $eventPlugins;
+    public $eventPlugins;
```

Comment:
If you use public properties, Doctrine can not create a proxy.
Please write a getter-method to access the property.